docs/governance.rst

```diff
@@ -0,0 +1,244 @@
+      the PR of which neither is the author of the PR. 
+      The reviews should be made after the last commit in the PR (equivalent to 
+      `Stale review dismissal <https://help.github.com/articles/enabling-required-reviews-for-pull-requests/>`_
+      option on GitHub). If a second review requests minor changes after
+      another reviewer approved the PR, the first review does not need
+      to re-review.
+   b) Does not feature any `Reviews that Request changes <https://help.github.com/articles/about-required-reviews-for-pull-requests/>`_.
+      That is, if someone asked for changes, the PR should not be merged just
+      because two other people approve it.
+   c) Is not a Draft PR. That is the PR author says it is ready for review.
+   d) Passes all automated tests.
+   e) Is not proposing a new release.
+   f) The steering committee has not added extra restrictions. For example, if
+      a PR is a non-trival change, the steering committee can create a system
+      to get feedback from more than just two reviewers before merging.
+5. After consultation with contributors, the steering committee can decide
+   to merge any PR - even if it's not eligible to merge according to Rule 4.
+6. Anyone can Review a PR and request changes. If a community
+   member requests changes they need to provide an explanation regarding what
+   changes should be made and justification of their importance. Reviews
+   requesting changes can also be used to request more time to review a PR.
+7. A reviewer who requested changes can dismiss their own review, if
+   they decide their requested changes are no longer necessary, or approve
+   changes that address the issue underlying their change request.
+8. If the author of a PR and a reviewer who requests changes cannot find a
+   solution that would lead to: (1) The author closing the PR without merging
+   (2) The reviewer accepting requested changes or (3) The dismissing their
+   review, so that the PR can be approved and merged, then the disagreement
+   will be resolved with a vote.
+9. Rules governing voting:
+
+   a) A vote can be triggered by any Maintainer, but only after 5 working days
+      from the time a Review Requesting Changes is made. If a PR has a disagreement
+      that required a vote, there must be at least 15 days from the conclusion
+      of the first vote before another vote can be triggered.
+   b) Only Maintainers can vote and each Maintainer gets one vote.
+   c) A vote ends after 7 working days or when all Maintainers have voted or
+      abstained (whichever comes first).
+   d) A vote freezes the PR - no new commits or Reviews Requesting Changes can
+      be added to it while a vote is ongoing. If a commit is accidentally made
+      during that period it should be reverted. Comments are allowed.
+   e) The quorum for a vote is five votes.
```

Comment:
Just flagging that this might be high for our community, though I agree that 3 feels too low ! If we go with 5, we'll definitely need the longer (e.g. 15 day) time period.

Reply:
My sense is, if a vote it's called, it's likely after several weeks of discussion and attempts to resolve a conflict without a vote. 7 feels reasonable to me, but my opinion on this isn't particularly strong.

Reply:
One other thing is that there's no good infrastructure for voting on GitHub, so it will always end up being a slow process. E.g., we don't have ways to automatically count votes, to set time limits on them, or to block pull requests based on the results of a vote. I think that's part of why BIDS has the voting system in their decision-making file, but it never gets used as described.

Reply:
If only Maintainers can merge pull requests and our maintainers are aware of these rules, we can block PRs during voting. Also, by definition, a vote will only happen if there is an unresolved comment from a reviewer, which would prohibit any random person from merging the PR.

I also like not describing the mechanics of voting in these rules because I think that flexibility will be useful. We know everyone who is allowed to vote, and, if we ever have a large enough number of voters such that a manual tally would be difficult, we'll have had to revise the bylaws well before that point. 

As for the practical mechanics, every vote could be an issue that lists the vote closing deadline, and people vote by commenting on that issue. I don't think we need to specify that in the docs, but it's just one option.

Reply:
I believe that with a group this size, getting lost in the mechanics could probably take a long time. My other fear is that if votes stretch out it will just go forgotten and we'll circle around endlessly. Since voting is (as you put in a comment above) a fall-back, I think it's reasonable to say that voting should be in reaction to a thorough discussion, rather than before one.

Reply:
I'm fine making a vote take 15 days & keeping this at 5. A standard review is 3 people. A vote-worthy disagreement means that you already have 3 votes. 5 votes means at least two more people need to look into an issue. That said, given my comment above, it would be nice to be able to cut a 15 day voting window short. I have thoughts on criteria to cut votes short, but they might not be worth the effort, since most things in tedana aren't THAT rushed.

Reply:
I think it's not worth the effort.